app/konnect/servicehub/index.md

```diff
@@ -1,16 +1,16 @@
 ---
-title: The Konnect Service Hub
+title: The Konnect Service Catalog
```

Comment:
Shouldn't this file be deleted, since it's replaced by `app/konnect/service-catalog/index.md`?